CHANGELOG.md

```diff
@@ -2,5 +2,10 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+<a name="0.0.3"></a>
```

Comment:
I guess `standard-version` should generate text here based on commit messages? I'll have to investigate that, I thought my commit messages would work here